README.md

```diff
@@ -36,7 +36,7 @@ The ultimate service for managing multi-channel notifications with a single API.
     ·
     <a href="https://github.com/novuhq/novu/discussions">Request Feature</a>
     ·
-  <a href="https://discord.gg/8KpBEjehEV">Join Our Discord</a>
+  <a href="https://discord.gg/novu">Join Our Discord</a>
```

Comment:
This breaks the Discord invite. Please rollback.

Reply:
Oh seems like my mistake :( I was sure this link works.

We can use created by @iampearceman 
```suggestion
  <a href="https://discord.novu.co">Join Our Discord</a>
```